src/tophu/_io.py

```diff
@@ -480,6 +480,7 @@ def __init__(
         driver: str | None = None,
         crs: str | dict | rasterio.crs.CRS | None = None,
         transform: rasterio.transform.Affine | None = None,
+        **options: dict[str, Any],
```

Comment:
Is there any convenient way to check the resulting ENVI/GeoTiff/whatever file to check that these options were properly applied? We need a unit test to exercise this.

Reply:
Sure I'll pull some from the dolphin tests. 
- we could check the common ones for GTiff; I don't think it's worth it/possible to check all possible options 
- The envi one is just "did it make the file `whatever.unw.hdr` or did it name it `whatever.hdr` (which I avoid because the unwrapping produces a bunch of files with similar names, and GDAL can get confused without fuller extensions)